src/test/coverartutils_test.cpp

```diff
@@ -34,13 +34,15 @@ class CoverArtUtilTest : public MixxxTest, public CoverArtCache {
 
 const QString kCoverLocationTest("res/images/library/cover_default.png");
 const QString kTrackLocationTest(QDir::currentPath() %
-                                 "/src/test/id3-test-data/cover-test.mp3");
+                                 "/src/test/id3-test-data/cover-test-png.mp3");
 
 TEST_F(CoverArtUtilTest, extractEmbeddedCover) {
     const QString kTestPath(QDir::currentPath() % "/src/test/id3-test-data/");
     QImage cover;
     // We never need to acquire security tokens for tests since we don't run
     // them in a sandboxed environment.
+	
+    // TODO(XXX) we need to compare with a reference file to ensure that we are reading correct data
```

Comment:
1. I used [easytag](https://wiki.gnome.org/Apps/EasyTAG) to embbed the covers. The image from the [mp3 is here](http://www.imagenetz.de/f3054e578/cc.png.html).
2. Well there is only one way to find out try. But I don't expect them to be different since metadata is specified in the ID3v2 specs and should be independent of the file format.
3. All the stuff in `searchImage` happens in a temporary directory. Look at line 76.
   If you have more questions about that just ask
4. Looks like this is needed to help work out the bugs ;). Thanks for the work